tracing-subscriber/src/filter/layer_filters/combinator.rs

```diff
@@ -319,30 +318,25 @@ where
         Some(cmp::max(self.a.max_level_hint()?, self.b.max_level_hint()?))
     }
     #[inline]
```

Comment:
nit: looks like there's a missing line of whitespace here
```suggestion
    }
    
    #[inline]
```